### PR TITLE
Adding Fabric MTU test for max configured P2P uplinks

### LINKED ISSUE #676: Test MTU with eos_validate_state on reachability tests
**Issue Type**

- [ ] Template enhancement
- [x] Role enhancement
- [ ] Documentation enhancement (avd.sh)

**Is your feature request related to a problem? Please describe.**

In eos_validate_state role we can test MTU when we are doing end to end pings for DC or Loopback0. This will help testing the MTU of the fabric.

**Describe the solution you'd like**

We can add `size:` and `df-bit` to the ping tests.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by xaviramon**
## Change Summary
Adding 2 new tests on ansible to generate a ping with df-bit for 9214 bytes. CSV results file is updated with MTU tests.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes)
- [x] Documentation content changes
- [ ] Other (please describe):

## Related Issue(s)

Fixes issue #676 

## Component(s) name

`arista.avd.validation`

## Checklist:

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://www.avd.sh/docs/contributing/) document.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly
- [x] All new and existing tests passed ([`pre-commit`](https://www.avd.sh/docs/installation/development/#python-virtual-environment), `make linting` and `make sanity-lint`).

### CODE REVIEW

**Review by ClausHolbechArista — APPROVED** [review]
LGTM

File: ansible_collections/arista/avd/roles/eos_validate_state/tasks/loopback0_reachability.yml [hunk 1]
```diff
@@ -29,3 +29,33 @@
...
+  ignore_errors: "{{ eos_validate_state_validation_mode_loose }}"
+  when: |
+    (loopback0_reachability.loopback0_range is arista.avd.defined) and
+    (loopback_interfaces.Loopback0.ip_address is arista.avd.defined) and
+    (type is defined and type != "l2leaf")
+  register: fabric_mtu_state
+  tags:
+    - loopback0_reachability
+
+- name: Validate MTU is {{ p2p_uplinks_mtu }} bytes between devices (loopback0 <-> loopback0)
```
Comment (carlbuchmann) [inline ansible_collections/arista/avd/roles/eos_validate_state/tasks/loopback0_reachability.yml:48]: Tests should all be build leveraging structure configuration data only i.e. not using `{{ p2p_uplinks_mtu }}
`, so they work across various use cases, including deployments not leveraging the eos_design role.

I would suggest doing this test on the physical P2P interfaces and leveraging the `ethernet_interfaces[interface].mtu`  value.

**Review by carlbuchmann — CHANGES_REQUESTED** [review]
I would recommend updating to do the MTU test with physical interfaces,
add MTU to `ip_reachability.yml` since the proposed implementation would only work if leveraging the eos_design role.

**Comment by carlbuchmann** [comment]
Closing PR as this has been stale for several months!

**Pull request closed without merging**